messaging/src/main/java/org/axonframework/eventhandling/processors/streaming/pooled/PooledStreamingEventProcessor.java

```diff
@@ -205,7 +205,7 @@ private String calculateIdentifier() {
         var unitOfWork = unitOfWorkFactory.create();
         return joinAndUnwrap(
                 unitOfWork.executeWithResult(context -> CompletableFuture.completedFuture(
-                        tokenStore.retrieveStorageIdentifier().orElse("--unknown--"))
+                        joinAndUnwrap(tokenStore.retrieveStorageIdentifier()).orElse("--unknown--"))
```

Comment:
Again not really a `TokenStore` concern, more an `PooledStreamingEventProcessor` concern. I don't think we need to use the `UnitOfWorkFactory` here, right? Or am I missing something obvious, @MateuszNaKodach?